examples/wiki/wiki.py

```diff
@@ -9,6 +9,10 @@
 WIKIPART = re.compile(r'([A-Z][a-z0-9_]+)')
 WIKIWORD = re.compile(r'([A-Z][a-z0-9_]+(?:[A-Z][a-z0-9_]+)+)')
 
+@app.route('/', methods=['GET'])
+def redirect():
+    return redirect(url_for('HomePage'))
```

Comment:
I'll mess with it a little more extensively later tonight but that should be it.